SocialCareCaseViewerApi.Tests/V1/Helpers/CaseStatusHelper.cs

```diff
@@ -26,16 +22,55 @@ public static class CaseStatusHelper
         public static (CaseStatus, SocialCareCaseViewerApi.V1.Infrastructure.Person) SavePersonWithCaseStatusToDatabase(
           DatabaseContext databaseContext)
         {
-            var caseStatusType = TestHelpers.CreateCaseStatusType();
-            var caseStatusSubtype = TestHelpers.CreateCaseStatusSubtype(typeId: caseStatusType.Id);
+            var caseStatusType = TestHelpers.CreateCaseStatusType(id: 10);
+            var caseStatusSubtype = TestHelpers.CreateCaseStatusSubtype(typeId: caseStatusType.Id, id: 20);
+            var person = TestHelpers.CreatePerson(3);
+            var csus = TestHelpers.CreateCaseStatus(personId: 3, typeId: caseStatusType.Id, subtypeId: caseStatusSubtype.Id, startDate: DateTime.Today, notes: "Testing");
```

Comment:
```suggestion
            var caseStatus = TestHelpers.CreateCaseStatus(personId: 3, typeId: caseStatusType.Id, subtypeId: caseStatusSubtype.Id, startDate: DateTime.Today, notes: "Testing");
```

More readable and explicit / I can't stop seeing "sus" and then thinking of Among Us... 😂 Same for the below method.